video/cloud-client/shotchange/shotchange.py

```diff
@@ -58,13 +56,14 @@ def analyze_shots(path):
     # [END check_operation]
 
     # [START parse_response]
-    shots = operation.result().annotation_results[0]
-
-    for note, shot in enumerate(shots.shot_annotations):
-        print('Scene {}: {} to {}'.format(
-            note,
-            shot.start_time_offset,
-            shot.end_time_offset))
+    shots = operation.result().annotation_results[0].shot_annotations
+
+    for i, shot in enumerate(shots):
+        start_time = (shot.start_time_offset.seconds +
+                      shot.end_time_offset.nanos / 1e9)
```

Comment:
As above. Might be worth making an offset_to_seconds helper function?

Reply:
that would make a lot of sense, but since we include code snippets in the devsite, refactoring out a helper function means that the user cannot use the code copied off the devsite unless they also locate and copy the helper functions.